ddlite.py

```diff
@@ -680,18 +679,18 @@ def __init__(self, candidates, feats=None, gt_dict=None):
     # LF data
     self.lf_matrix = None
     self.lf_names = []
-    # Model data    
-    self.X = None
-    self._w_fit = None
-    self.w = None
+    # Model probabilities
+    self.model = None
+    self.probabilities = None
+    self.lf_marginals = None
+    self.lf_weights = None
```

Comment:
Still a little confused here- what's `self.probabilities`?

I think `lf_marginals` should maybe be `training_marginals` or something (otherwise sounds like it's the marginal prob of the LFs)